### DISCUSSION TITLE
Are channel discovery and subscriptions not anonymized?

### DISCUSSION THREAD

**Discussion by da2x** [Tribler/tribler#5607] [Q&A] [answered]
I'm uncertain about the implication of commit a5c10f784945163d848399e0baaecf1b57aa035a by @ichorid. There's no commit message with the change so I don't quite understand what the change does.

Did channel subscriptions, discovery, retrieval, etc. used to be anonymized? and this change made them non-anonymized?

**Comment by qstokkink** [comment]
They used to follow the default download configuration, see:

https://github.com/Tribler/tribler/blob/fecebd68eb0609016711bf8bdd2ec9d93f0f5a07/src/tribler-core/tribler_core/modules/libtorrent/download_manager.py#L409

This means that if for example if your default setting was to download with 2 hops, channels would also download with 2 hops of anonymity.

Now they are indeed never anonymized.

**Comment by da2x** [comment]
> Now they are indeed never anonymized.

Why not? What was the motivation for the change? To me, this seems like an odd choice in a project that strives to provide people with more anonymity.

**Comment by ichorid** [comment, marked as answer]
See https://github.com/Tribler/tribler/issues/4982
